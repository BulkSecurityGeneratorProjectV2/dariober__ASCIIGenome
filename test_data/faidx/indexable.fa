

>regular;with_comment
ACTGACTGACTGACTGACTGACTGACTGACTG
ACTGACTGACTGACTGACTGACTGACTGACTG
ACTGACTGACTGACTGACTGACTGACTGACTG
ACTGACTGACTGACTGACTGACTGAC
>sequence_in_one_line
ACTGACTGACTGACTGACTGACTGACTGACTGACTGACTGACTGACTGACTGACTGACTGACTG
>>name_starts_with_>
NNNNNNNNNNNNNNNNNN


>1_with	tab	in	name
ACTGACTGACTGACTGACTGACTGACTGACTG
ACTGACTGACTGACTGACTGACTGACTGACTG
ACTGACTGACTGACTGACTGACTGACTGACTG
ACTGACTGACTGACTGACTGACTGAC
>2_with space in name
ACTGACTGACTGACTGACTGACTGACTGACTG
ACTGACTGACTGACTGACTGACTGACTGACTG
ACTGACTGACTGACTGACTGACTGACTGACTG
ACTGACTGACTGACTGACTGACTGAC
> space_after_>   
ACTGACTGACTGACTGACTGACTG
>triplets
ACT GAC TGA CTG ACT GAC TGA CTG ACT GAC
ACT GAC TGA CTG ACT GAC TGA CTG ACT GAC
ACT GAC TGA CTG ACT GAC TGA CTG ACT GAC
ACT GAC TGA CTG ACT GAC
>Trailing_space_in_sequence
ACTGACTGACTGACTGACTGACTG   
ACTGACTGACTGACTGACTGACTG   
ACTGACTGACTGACTGACTGACTG   
NNNN    


