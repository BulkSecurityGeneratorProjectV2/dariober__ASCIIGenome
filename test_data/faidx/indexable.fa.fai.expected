regular;with_comment	122	24	32	33
sequence_in_one_line	64	172	64	65
>name_starts_with_>	18	258	18	19
1_with	122	299	32	33
2_with	122	447	32	33
space_after_>	24	592	24	25
triplets	108	627	30	40
Trailing_space_in_sequence	76	799	24	28
